src/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class Main {
+
+    public static int sumOfThree(int num) {
+        int first = num / 100;
+        int second = (num - (first * 100)) / 10;
+        int third = num - (first * 100 + second * 10);
+        return first + second + third;
+    }
+
+    public static int sumOfThree(String a, String b) {
+        int w1 = a.length();
+        int w2 = b.length();
+        return w1 + w2;
+    }
+
+    public static boolean sumOfThree(String a) {
```

Comment:
Number should go from number inputted via keyboard (you need to use Scanner) in the middle of the method. I see you used it separately from the method